### DISCUSSION TITLE
[Suggestion] Dark Auction Items Bought Count

### DISCUSSION THREAD

**Discussion by doej1367** [HypixelDev/PublicAPI#272] [Ideas]
Woudn't it be cool if you could see how many dark auction items someone already bought?
I would really like to see how many I bought, because then I would know how many I need to buy to get the first DA talisman.
Maybe a new stat called 'dark_auction_milestone_items_bought'?

**Comment by vyketype** [comment]
Yes, that would be very nice indeed.

**Comment by TheProGuru** [comment]
This would be extremely useful for people trying to get Seal of the family that are unsure of how many dark auction items they have bought
